src/main/java/org/spongepowered/api/potion/PotionEffectType.java

```diff
@@ -35,10 +36,18 @@
 public interface PotionEffectType extends CatalogType, Translatable {
 
     /**
-     * Gets whether this potion effect is applied
-     * instantly or over time.
+     * Gets whether this potion effect is applied instantly or over time.
      *
      * @return If applied instantly.
      */
     boolean isInstant();
+
+    /**
+     * Gets the {@link Translation} for this potion effect type as a potion
+     * name.
+     *
+     * @return The translation representing this effect as potion
+     */
+    Translation getPotionTranslation();
```

Comment:
How is this different from the existing translation defined by the Translatable type? In other words, I don't recall potion effects having two separate translations